README.md

```diff
@@ -1,30 +1,34 @@
 linter-flake8
 =============
 
-linter-flake8 is a [flake8](https://pypi.python.org/pypi/flake8 provider for [Linter](https://github.com/AtomLinter/Linter).
+linter-flake8 is a [flake8](https://pypi.python.org/pypi/flake8) provider for [Atom-Linter](https://github.com/AtomLinter/Linter).
 
 ![img](https://cloud.githubusercontent.com/assets/4278113/8768482/52f975c6-2e3f-11e5-87e4-27c8359fd36c.gif)
 
 ### Requirements
-Linter package must be installed in order to use this plugin. If it's not
-installed, please follow the instructions [here](https://github.com/AtomLinter/Linter).
+The `Linter` package must be installed in order to use this package. If it's not installed, please follow the [Linter instructions](https://github.com/AtomLinter/Linter).
```

Comment:
\[installation instructions\] instead? They aren't the instructions on how to lint.

Also you might as well link directly to the installation instructions: https://github.com/atom-community/linter#how-to--installation